scripts/m3dbnode_bootstrapped.sh

```diff
@@ -48,14 +48,26 @@ if [ "$RES" -ne 0 ]; then
   exit 0
 fi
 
-curl -sSf "$COORD_NAMESPACE_ENDPOINT" >/dev/null
+curl -sSf -o "$COORD_TMPFILE" "$COORD_NAMESPACE_ENDPOINT"
 RES=$?
 
 if [ "$RES" -eq 22 ]; then
   echo "Received 4xx from namespace endpoint $COORD_NAMESPACE_ENDPOINT"
   exit 0
 fi
 
+# If there IS a placement but NO namespace, then the dbnode will respond
```

Comment:
Hm how does this distinguish from the time when there is a namespace in etcd but it hasn't yet been picked up by the node at startup?

Reply:
Good point, I hadn't thought of that. Any advice to get around the issue without waiting for #996?